mlflow/entities/trace_location.py

```diff
@@ -215,8 +243,19 @@ def from_experiment_id(cls, experiment_id: str) -> "TraceLocation":
         )
 
     @classmethod
-    def from_uc_schema(cls, catalog_name: str, schema_name: str) -> "TraceLocation":
+    def from_uc_schema(
```

Comment:
Can we include "databricks" in the name, e.g `from_databricks_uc_schema`? Also, can we avoid placing these databricks-specific converts on entities? It's a bit odd for OSS users to see public methods like `from_uc_schema`.